[ADD] subscriptions: add base documentation for Subscriptions (#617)
odoo/documentation-user: subscriptions/configuration/subscription_templates.rst

==========================
Use subscription templates
==========================

The subscription business model is becoming more and more popular. Are you wondering why? Because a
subscription model simplifies the business process and makes it very easy for both customers and the
business. It ensures that the business retains more customers on a consistent basis and that clients
can be reached out to and engaged with regularly. A business model like this makes it easy to
anticipate demand and supply. Finally, this model offers much higher payment security for your
business.

.. raw:: html

   <div align="center" style="color:#AD5E99; font-size: 2rem ;margin: 20px 0"> <b>Simplicity.
   Higher customer retention. Opportunities for marketing. Business consistency. Better cash flow
   management.</b> </div>

> Odoo subscription templates will help you save a lot of time and money. Indeed, these templates will
> help you generate recurring invoices and manage renewals at a fast pace. With Odoo you have the
> possibility to create, edit and manage your own subscription templates.

Reviewer: ```suggestion
Using templates allows you to save time as they allow you to generate recurring invoices and manage renewals at a glance, and keep consistency, as you do not need to create individual forms for every transaction.
```

It has to be improved of course, but you might get my suggestion. A suggestion that could be used as a base to a brainstorm (:

Configuration
=============

> Go to :menuselection:`Subscriptions → Configuration → Subscription templates`. By default, Odoo
> suggests you two types of subscription (MON - Monthly subscription *vs* YEA - Yearly subscription).
> Of course, you can create your own ones.

Reviewer: ```suggestion
Go to :menuselection:`Subscriptions --> Configuration --> Subscription Templates`. By default, Odoo
suggests you two types of subscription (MON - Monthly subscription *vs* YEA - Yearly subscription), 
or create your own ones.
```

.. image:: media/subscription_templates_1.png
  :align: center

>   :alt: How to use subscription templates on Odoo Subscriptions?

Reviewer: Reminder about the alt tags: we actually use them to describe the image, with a few words, as if we would for a blind person (:

.. important::

>    Installing the **Subscriptions** application will automatically install the **Sales** and
>    **Invoicing** applications.

Reviewer: ```suggestion
   The **Subscriptions** application automatically installs **Sales** and
   **Invoicing** as they work integrated.
```

Create your first template
==========================

> You can create a new template or edit an existing one. The first thing you need to do is give your
> template a name. After that, you have to choose an invoicing period. It is possible to change the
> amount and specify whether it is per days, weeks, months or years. The duration will determine if
> the subscription will go on forever (until it’s manually closed) or for a fixed amount of time.
> There are also several options for the payment mode. Something important here is that an additional
> field appears when you choose “Send”, “Send & try to charge” and “Send after successful payment”.
> This field gives you the possibility to add an invoice email template (as you can see below).

Reviewer: ```suggestion
You can create a new template or edit an existing one. The first thing you need to do is give your
template a name. After that, choose an *Invoicing Period* and specify whether you would like to invoice your customers per *Days*, *Weeks*, *Months* or *Years*. On *Duration* determine if
the subscription must go on *Forever* (until it’s manually closed), or for a *Fixed* amount of time.
Among the payment options, an additional field called *Invoice Email* appears when you choose *Send*, *Send & try to charge* or *Send after successful payment*. This field allows you to add an invoice email template.
```

as always, but I always state: suggestion. I like to use italic to make it clear for the user that those are the name of the options you will see on the app. I believe it makes the reading and understanding easier (:
Keep in mind that this explanation is valid for all comments that I might do such suggestion ^.^

.. image:: media/subscription_templates_2.png
  :align: center
  :alt: How to use subscription templates on Odoo Subscriptions?

> On each template, you can also configure some options for the customer portal. First of all, you are
> going to be able to say if you want the customer to be able to close the subscription or not. After
> that, you have the automatic closing limit, the group of subscription and the journal options that
> you can also manage directly from the subscription templates.

Reviewer: ```suggestion
For each template, you can also choose if you want the customer to be able to close their subscription or not. If enabled (I'm guessing this hehe), you can set an *Automatic Closing* limit, the *Group of Subscription* and the *Journal* options. You can also manage these options directly from the subscription templates.
```

I feel like these features 'automatic, group, journal' could have a little explanation? no? I believe that this is the reason we mention features in text, to explain them, at least most of the time.

Author: It is a link with the Accounting app. If I give an explanation for the three of them, it will be too long I guess. Corentin, the ex PO of subscriptions, told me that it is not necessary to explain these fields :/

Reviewer: ohh ok!

.. note::

>    For each template, you can add your **Terms and Conditions** and your **Health Check**.

Reviewer: Would not it be nice to actually have a couple of titles explaining these tabs?

.. important::
   After creating your own subscription template, be sure to check out our documentation on how to
   create, edit and manage your own subscription products: :doc:`subscription_products`, to complete
   the sales flow.

.. seealso::

>    - :doc:`subscription_products`

Reviewer: As you added the seealso I would remove the paragraph. I believe the seealso is enough and it does the job of guiding the user through the flow (:

Author: I know this is repetitive but I like to put the "see also" if I mentioned a doc in the text :) It is two times but it is necessary for some customers ^^

==========================
Use subscription templates
==========================

The subscription business model is becoming more and more popular. Are you wondering why? Because a
subscription model simplifies the business process and makes it very easy for both customers and the

> business. It ensures that the business retains more customers on a consistent basis and that clients

Reviewer 2: Why does it make it easy? 

It's still not a good pitch. some ideas here: https://www.entrepreneur.com/article/243573

can be reached out to and engaged with regularly. A business model like this makes it easy to
anticipate demand and supply. Finally, this model offers much higher payment security for your

> business.

Reviewer 2: Some repetition, I'd rewrite it 

Key point: Recurring revenue, ease of process for customers

some good ideas in this: 

For customers, the value lies in the convenience. First, there’s the autopilot simplicity of subscriptions that removes the thinking out of a purchase decision. Subscribers never have to remember to reorder every month, which gives them the reassurance that they will have whatever they need before they actually need it. Everything arrives to your door like magic -- removing the hassle of making a trip to the store or website to place an order. Second, subscriptions offer a flat rate which helps customers stay within their budget -- always. Lastly, subscriptions usually bring added value to the customer through bundling or getting it all for the price of one.

For businesses, the value of a subscription is the ability to predict revenue through recurring sales. In fact, according to John Warrillow, creator of The Value Builder System, recurring revenue is perhaps one of the most compelling factors in a company valuation. “The more guaranteed revenue you can offer a potential acquirer, the more valuable your business is going to be," Warrillow says. "Because a high percentage of the revenue of a subscription-based business is recurring, its value will be up to eight times that of a comparable business with very little recurring revenue.”

==========================
Use subscription templates
==========================

The subscription business model is becoming more popular. Are you wondering why?

- For **customers**, value lies in **convenience**:

>   - Subscriptions simplify the business process. Indeed, subscribers never have to remember to renew
>     their orders every month, which gives them the assurance that they will have everything they
>     need before they actually need it.
>   - Subscriptions help customers stay on budget.
>   - Subscriptions bring added value to the customer.

Reviewer: ```suggestion
  - Subscriptions simplify the business process. 
  - Subscribers do not need to remember to renew their orders every month, which assures they will have what 
     they need before they realize they need it.
  - Subscriptions help customers stay on budget.
```
- Subscriptions bring added value to the customer. <- the values are what the sentences describe?! So, I believe this sentence could be removed (:

- For **businesses**, value lies in the **ability to predict recurring revenue**:

>   - Subscriptions reduce customer churn rate and significantly increase customer retention.
>   - Subscriptions provide much higher payment security for your business. They stabilize and
>     maintain recurring revenue streams. Indeed, the more guaranteed revenue you can offer, the more
>     valuable your business will be.

Reviewer: ```suggestion
  - Subscriptions reduce customer churn rate and significantly increase customer retention.
  - Subscriptions provide higher payment security for your business. They stabilize and
    maintain recurring revenue streams, guaranteeing monthly revenues, and adding value to your business.
```

 the more guaranteed revenue you can offer <- What does that mean? ^^

.. raw:: html

   <div align="center" style="color:#AD5E99; font-size: 2rem ;margin: 20px 0"> <b>Simplicity.
   Higher customer retention. Opportunities for marketing. Business consistency. Better cash flow
   management.</b> </div>

> Odoo subscription templates will help you save a lot of time and money. Indeed, these templates will
> help you generate recurring invoices and manage renewals at a fast pace. With Odoo you have the
> possibility to create, edit and manage your own subscription templates.

Reviewer: ```suggestion
Odoo subscription templates help you save time and money. Templates can
help you generate recurring invoices and manage renewals at a fast pace. With Odoo you have the
possibility to create, edit, and manage your own subscription templates.
```

From the book Style, he says : 'don’t modify absolute words'. That is why throughout the docs I review I suggest remove such words (:

Configuration
=============

> Go to :menuselection:`Subscriptions → Configuration → Subscription templates`. By default, Odoo
> suggests you two types of subscription (MON - Monthly subscription *vs* YEA - Yearly subscription).
> Of course, you can create your own ones.

Reviewer: ```suggestion
Go to :menuselection:`Subscriptions --> Configuration --> Subscription templates`. By default, Odoo
suggests you two types of subscription (MON - Monthly subscription *vs* YEA - Yearly subscription).
You can also create your own ones.
```

.. image:: media/subscription_templates_1.png
  :align: center
  :alt: Default subscription templates on Odoo Subscriptions

.. important::
   The **Subscriptions** application automatically installs **Sales** and **Invoicing** as they work
   integrated.

Create your first template
==========================

> You can create a new template or edit an existing one. The first thing you need to do is give your
> template a name. After that, choose an *Invoicing period* and specify whether you would like to
> invoice your customers per *Days*, *Weeks*, *Months* or *Years*. On *Duration* determine if
> the subscription must go on *Forever* (until it’s manually closed) or for a *Fixed amount* of time.
> Among the payment options, an additional field called *Invoice email* appears when you choose
> *Send*, *Send & try to charge* or *Send after successful payment*. This field allows you to add an
> invoice email template to your subscription templates.

Reviewer: ```suggestion
You can create a new template or edit an existing one. The first thing you need to do is give your
template a name. After that, choose an *Invoicing period* and specify whether you would like to
invoice your customers per *Days*, *Weeks*, *Months* or *Years*. On *Duration*, determine if
the subscription must go on *Forever* (until it’s manually closed), or for a *Fixed amount* of time.
Among the payment options, an additional field called *Invoice email* appears when you choose
*Send*, *Send & try to charge* or *Send after successful payment*. This field allows you to add an
invoice email template to your subscription templates.
```

.. image:: media/subscription_templates_2.png
  :align: center
  :alt: Create your own subscription templates on Odoo Subscriptions

> For each template, you can also choose if you want your customers to be able to close their
> subscriptions or not. If enabled, you can set an *Automatic closing* limit and you can specify the
> *Group of subscription* and the *Journal* options.

Reviewer: ```suggestion
For each template, you can also choose if you want your customers to be able to close their
subscriptions or not. If enabled, you can set an *Automatic closing* limit and specify the
*Group of subscription* and *Journal* options.
```

.. note::
   On each template, you can add your **Terms and Conditions**. Specifying terms and conditions is
   essential to set out important contractual points between the customers and the sellers (payment,
   refund policy, cancellation, complaints, etc.).

   .. image:: media/subscription_templates_3.png
     :align: center
     :alt: Terms & conditions on Odoo Subscriptions

>    Finally, if you want to know the basic running health status of your subscriptions, you also have
>    access to a specific tab called **Health Check**. There, you can edit and create your own
>    filters to define what is a subscription in good health *vs* bad health. The system will
>    automatically summarize all the records corresponding to these filters and you will be able to
>    manage them in one click.

Reviewer: ```suggestion
   Finally, if you want to know the basic running health status of your subscriptions, you also have
   access to a specific tab called **Health Check**. There, you can edit and create your own
   filters to define what is a subscription in good health *vs* bad health. The system automatically summarizes all the 
   records corresponding to these filters and you are able to manage them in one click.
```

   .. image:: media/subscription_templates_4.png
     :align: center
     :alt: Health check on Odoo Subscriptions

.. important::

>    After creating your own subscription templates, be sure to check out our documentation on how to
>    create, edit and manage your own subscription products: :doc:`subscription_products`, to complete
>    the sales flow.

Reviewer: ```suggestion
   After creating your own subscription templates, be sure to check out our documentation on :doc:`how to
   create, edit, and manage your own subscription products <subscription_products>`, to complete the sales flow.
```
